Unix/common/windows/config.h

```diff
@@ -34,7 +34,6 @@
 #endif
 
 #define CONFIG_HAVE_FUNCTION_MACRO 1
-#define CONFIG_ENABLE_WCHAR 1
 #define CONFIG_HAVE_ATOMIC_INTRINSICS 1
 #define CONFIG_HAVE_WCSTOLL 1
```

Comment:
You probably want to also get rid of CONFIG_HAVE_WCSTOLL, CONFIG_HAVE_WCSCASECMP, WCSLEN, CONFIG_HAVE_WCSDUP also